.bumpversion.cfg

```diff
@@ -1,8 +1,7 @@
 [bumpversion]
-current_version = 2.3.1
+current_version = 3.0.0
```

Comment:
I think it makes sense to keep version bumping out of pull requests. The `version2bump` command already handles this change, and also tags the release, so to use `version2bump` the person handling the next release would actually need to undo this change locally before running the command, otherwise `versin2bump major` would set the version to 4.0.0 instead of 3.0.0.